content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -995,6 +995,19 @@ to record the version of Windows Server in use.
 
 The label's value is in the format "MajorVersion.MinorVersion.BuildNumber".
 
+### storage.alpha.kubernetes.io/migrated-plugins {#storagealphakubernetesiomigrated-plugins}
+
+Type: Annotation
+
+Example:`storage.alpha.kubernetes.io/migrated-plugins:" kubernetes.io/cinder"`
+
+UseT on: CSINode
+
+This annotation is automatically added when CSIMigration is turned on for the CSINode object that maps to a node that installs CSIDriver. This annotation shows the in-tree plugin name of the migrated plugin. Its value depends upon the in-tree cloud provider storage type that is used in your cluster.
+
+For example, if we enable the migration where the in-tree cloud provider storage type is `CSIMigrationvSphere`, the CSINodes instance for the node should be updated with:
```

Comment:
[Avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we).

Reply:
I'll keep that in mind .